cmd/lock-rpc-server.go

```diff
@@ -269,11 +269,10 @@ func (l *lockServer) lockMaintenance(interval time.Duration) {
 			// - some network error (and server is up normally)
 			//
 			// We will ignore the error, and we will retry later to get resolve on this lock
-			log.Println("  Dsync.Active failed:", err)
+			log.Infoln("Dsync.Active failed:", err)
```

Comment:
I'll remove the logging line then because the user really shouldn't be interested in the actual error.